addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -106,6 +106,54 @@
         </t>
     </t>
 
+    <!-- Inline Boolean Field -->
+    <t t-name="website.form_field_inlinebool">
+        <div t-attf-class="s_website_form_field mb-3 #{field.formatInfo.col or 'col-12'} #{field.custom and 's_website_form_custom' or ''} #{(field.required and 's_website_form_required' or '') or (field.modelRequired and 's_website_form_model_required' or '')} #{field.hidden and 's_website_form_field_hidden' or ''} #{field.dnone and 's_website_form_dnone' or ''}"
```

Comment:
Too much redundant code, can't we just use the existing `website.form_field` layout here(may be with some changes)?